internal/elasticsearch/client.go

```diff
@@ -75,17 +76,27 @@ type DeleteResult struct {
 }
 
 func NewClient(httpClient HTTPClient, logger *logrus.Logger) (*Client, error) {
+	region := os.Getenv("AWS_REGION")
+	if region == "" {
+		region = "eu-west-1"
+	}
+
+	sess, err := session.NewSession(&aws.Config{Region: aws.String(region)})
+	if err != nil {
+		panic(err)
+	}
```

Comment:
Can use `session.Must` to let session do the error handling:
```suggestion
	sess := session.Must(session.NewSession(&aws.Config{Region: aws.String(region)}))
```

Reply:
That did work when I tried it the other morning, but it could have been something else messing with it.

I'll run another test to double check it .